src/main/java/app/bpartners/geojobs/endpoint/event/model/AnnotationTaskRetrievingSubmitted.java

```diff
@@ -0,0 +1,43 @@
+package app.bpartners.geojobs.endpoint.event.model;
+
+import app.bpartners.geojobs.endpoint.event.EventStack;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Duration;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Data
+@EqualsAndHashCode(callSuper = false)
+@ToString
+public class AnnotationTaskRetrievingSubmitted extends PojaEvent {
+  @JsonProperty("jobId")
+  private String jobId;
+
+  @JsonProperty("firstAnnotationJobId")
+  private String firstAnnotationJobId;
```

Comment:
I think this is bad. Because now, you're statically giving the annotation jobs ids. And here, if there is 3 annotation jobs associated to the machine detection job, it's not possible to do it, if you do not add a `thirdAnnotationJobId`.

An other way, is to just give the jobId that can retrieve the end-to-end annotationJobIds that correspond dynamically, if it's two then it will be two, if three, etc. Same inside the service and the consumer, so it will handle dynamically the retrieve of the annotationTask associate to (potential) some annotations jobs associated to the same detectionJob.